lib/board.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    self.reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, ' ')
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts '-----------'
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts '-----------'
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(position)
+    cells[position.to_i - 1]
+  end
+
+  def full?
+    true if cells.include?(' ') == false
```

Comment:
Prefer implicit return values of boolean statements.

`cell.include?(' ') == false`

Or, going further, avoid equality comparison with boolean values. Here you can use the bang operator `!` to negate the results.

`!cells.include?(' ')`